spec/support/backend_examples.rb

```diff
@@ -33,3 +33,15 @@
     end
   end
 end
+
+RSpec.shared_context "with mocked request" do |path:, params:|
+  before do
+    request = double("request") # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
(opt) `rubocop:[disable|enable]` could be applied at the block level (either on `before` or `shared_context`. OK with keeping it inline to ensure that it's not automatically applied on future changes.

Reply:
I'm aware, but I wanted to make this a special case. I don't want to add a dependency on Rails to get the real request.

Reply:
Now that I've added Rack as a dependency I don't need to use an unverified double :+1: